pkg/roachprod/install/cluster_synced.go

```diff
@@ -2597,24 +2597,33 @@ func (c *SyncedCluster) Get(
 		close(results)
 	}()
 
+	var errOnce sync.Once
```

Comment:
I don't think this is necessarily used in a concurrent access context (just a regular for loop on a channel, unless I missed something), so you might be able to get away with just checking if the value has been set already, if that is the intention (see questionary comment below)

Reply:
Yeah that sounds right to me, removing the `sync`.